src/poll.rs

```diff
@@ -1289,7 +1754,7 @@ fn release_node(ptr: *mut ReadinessNode) {
         // Because `fetch_sub` is already atomic, we do not need to synchronize
         // with other threads unless we are going to delete the object. This
         // same logic applies to the below `fetch_sub` to the `weak` count.
-        if (*ptr).ref_count.fetch_sub(1, Release) != 1 {
+        if (*ptr).ref_count.fetch_sub(1, AcqRel) != 1 {
```

Comment:
Yeah, the `Release` is correct (I sure as hell hope so, because it is what `Arc` does), but this change makes the new thread sanitizer lint happy.